app/services/candidate_interface/measure_distances.rb

```diff
@@ -20,8 +18,10 @@ def distance(from, to)
       )
     end
 
+  private
+
     def has_coordinates?(model)
-      !model.latitude.nil? && !model.longitude.nil?
+      model && !model.latitude.nil? && !model.longitude.nil?
```

Comment:
Can use `model.geocoded?` I think!

Reply:
@malcolmbaig Yes, thanks! That's much tidier. I had to add the `geocoded_by` declaration to `Site`. Does that interfere with the syncing from the Find API though?

Reply:
Good question. We aren't invoking the `geocode` method on a site anywhere so arguably fine. Might be nice to override `Site#geocode` so that it raises a helpful error message instead (ie - stating the we get lat/long from Find).